text/0119-instance-options.md

```diff
@@ -96,7 +96,7 @@ The expected behavior of each flag is described in the list above.
 - [HTTP address & port binding](#333-http-address--port-binding)
 - [Master key](#334-master-key)
 - [Disable analytics](#335-disable-analytics)
-- [Dumps destination](#336-dumps-destination)
+- [Dump destination](#336-dump-destination)
```

Comment:
```suggestion
- [Dump directory](#336-dump-directory)
```

Reply:
Thanks, @gmourier! I checked for "dump" everywhere this time so there shouldn't be any other ... Apologies for the incomplete update!